Source/NSubstitute/Core/CallResults.cs

```diff
@@ -6,12 +6,18 @@ namespace NSubstitute.Core
     public class CallResults : ICallResults
     {
         readonly ICallInfoFactory _callInfoFactory;
+        private readonly bool _skipVoidMethods;
         ConcurrentQueue<ResultForCallSpec> _results;
 
-        public CallResults(ICallInfoFactory callInfoFactory)
+        public CallResults(ICallInfoFactory callInfoFactory, bool skipVoidMethodsForPerformance)
```

Comment:
Of course, I can do that, just don't like idea to copy-paste the whole class to change a single place. If later you fix some bug in this class, AF will not be updated :(

What if I rename [ReturnsVoidFrom](https://github.com/Zvirja/NSubstitute/blob/ea566ae65422ef768f02a7b2d5f6215d6b31a39e/Source/NSubstitute/Core/CallResults.cs#L47) method to `bool SkipVoidMethod()` and make it protected virtual? Will you accept such idea?.. 😊